### ISSUE TITLE
Allow additions to PERL_MB_OPT

### ISSUE THREAD

**Issue by djerius** [skaji/cpm#182]
Some distributions have custom `Module::Build` options that need to be applied, but `cpm` overrides the `PERL_MB_OPT` variable.

For example, `Params::Validate` v1.29 doesn't recognize `--pureperl_only`, but does recognize `--pp`.

I'd like to be able to do something equivalent to this:

`PERL_MB_OPT=--pp cpan  Params::Validate`

Thanks,
Diab

**Comment by skaji** [comment]
For `--pureperl-only`, it is defined in The Lancaster Consensus.

**Comment by djerius** [comment]
Agreed, it's just that there are older modules which don't support it yet, so the ability to add options to Build.PL would be useful.
